cypress/support/component.ts

```diff
@@ -32,38 +32,11 @@ window._oc_capabilities = { files: {} }
 
 // Example use:
 // cy.mount(MyComponent)
-Cypress.Commands.add('mount', (component, optionsOrProps) => {
```

Comment:
See Ferdinand's comment on this: https://github.com/nextcloud/server/pull/48725#discussion_r1812657604